src/list.js

```diff
@@ -0,0 +1,137 @@
+/* eslint-disable */
+import 'lodash';
+import dots from '../assets/images/dots.png';
+import removeIcon from '../assets/images/delete.png';
+import { MakeComplete } from '../modules/makeComplete.js';
+
+class DoList {
+  constructor(description, completed = false, index) {
+    this.description = description;
+    this.completed = completed;
+    this.index = index;
+  }
+}
+// Remove Function
+function removeTask(e) {
+  const itemClass = e.target.className;
+  const itemId = itemClass.split('-');
+  const id = parseInt(itemId[1], 10);
+  let getData = JSON.parse(localStorage.getItem('dolist'));
+  getData = getData.filter((task, index) => index !== id);
+  e.target.parentNode.remove();
+  getData.forEach((task, i) => task.index = i + 1);
+  localStorage.setItem('dolist', JSON.stringify(getData));
+  displayTask();
+};
+// Update Function
+const UpdateTask = (e) => {
+  const taskid = e.target.id;
+  const itemId = taskid.split('-');
+  const id = parseInt(itemId[1], 10);
+  const getData = JSON.parse(localStorage.getItem('dolist'));
+  const input = document.querySelector(`#activity-${id}`);
+  getData[id].description = input.value;
+  localStorage.setItem('dolist', JSON.stringify(getData));
+  displayTask();
+};
+// Edit function
+const editTask = (e) => {
+  const editItem = e.target;
+  const editBtn = document.getElementById(`${editItem.id}`);
+  const itemId = document.querySelector(`#activity-${editItem.id}`);
+  itemId.classList.add(`activity-${editItem.id}`);
+  itemId.id = `activity-${editItem.id}`;
+  itemId.removeAttribute('readonly');
+  itemId.focus();
+  const bgColor = document.querySelector(`#task-${e.target.id}`)
+  bgColor.style.backgroundColor = 'rgb(235, 235, 235)';
+  editBtn.style.display = 'none';
+  itemId.style.outlineColor = 'white';
+  itemId.style.backgroundColor = 'rgb(235, 235, 235)';
+  const inputFeild = document.querySelector('.item');
+  inputFeild.style.border = 'none';
+  const taskList = document.querySelector(`#tasks-${editItem.id}`);
+  taskList.classList.add('active');
+  const singleTask = document.querySelector(`#task-${editItem.id}`);
+  const removeBtn = document.createElement('img');
```

Comment:
I would suggest declaring all the variables once at the start of the function to make the code look cleaner.